.circleci/config.yml

```diff
@@ -12,13 +19,37 @@ jobs:
       - run:
           name: Install dependencies
           command: |
+            sudo apt-get update
+            sudo apt-get install cmake
             gem update --system
             gem install bundler
-            bundle install --path vendor/bundle
+            bundle install --path vendor/bundle --with development test
       - run:
           name: Running checks
           command: |
             git config --global user.email "circle@example.com"
             git config --global user.name "Circle CI"
             bundle exec overcommit -s
             bundle exec overcommit -r
+  test:
+    docker:
+      - image: circleci/ruby:2.5.5-stretch-node
```

Comment:
Please share all the same actions in &defaults, &references blocks. Also please save bundle cache and install bundler version from Gemfile.lock